.styleguidist/styleguidist.sections.js

```diff
@@ -93,6 +93,13 @@ module.exports = {
               name: 'Text',
               components: getComponents(['Text', 'Paragraph']),
             },
+            {
```

Comment:
Is this a primitive or more of a module?

Reply:
Moved to under modules.